base/setup/usetup/usetup.c

```diff
@@ -663,6 +663,15 @@ LanguagePage(PINPUT_RECORD Ir)
                 /* Load the font */
                 SetConsoleCodePage();
 
+                /* Update video mode */
+                MUIUpdateVideoMode();
```

Comment:
This should be part of the `SetConsoleCodePage` call, see comment above.

Reply:
OK. Fixed.